### PR TITLE
feat: convert core to a hybrid service

### PR DESCRIPTION
**Pull request by 0c370t**

### CODE REVIEW

**Review by gankoji — APPROVED** [review]

File: core/src/identity/identity.controller.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    async getUserByAuthAccount(@Payload() payload: unknown): Promise<GetUserByAuthAccountResponse> {
+        const data = CoreSchemas.GetUserByAuthAccount.input.parse(payload);
+        const user = await this.identityService.getUserByAuthAccount(
+            data.accountType as UserAuthenticationAccountType,
+            data.accountId,
+        );
+        const profile = await this.userService.getUserProfileForUser(user.id);
+        return {
+            id: user.id,
+            roles: user.type,
```
Comment (bwpikaard) [inline core/src/identity/identity.controller.ts:28]: If this is type, why is it roles? Inconsistency.

File: core/src/organization/organization.module.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ import {
     MemberRestrictionResolver, MemberRestrictionService, QueueBanGuard,
 } from "./member-restriction/";
 import {OrganizationResolver, OrganizationService} from "./organization";
+import {OrganizationController} from "./organization/organization.controller";
```
Comment (bwpikaard) [inline core/src/organization/organization.module.ts:12]: Barrel file?

File: core/src/organization/organization/organization.controller.spec.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import type {TestingModule} from "@nestjs/testing";
```
Comment (bwpikaard) [inline core/src/organization/organization/organization.controller.spec.ts:1]: Delete this? We don't have test files for any other controller, service, or resolver.

File: core/src/organization/organization/organization.controller.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import {MessagePattern, Payload} from "@nestjs/microservices";
+import {CoreEndpoint, CoreSchemas} from "@sprocketbot/common";
+
+import type {OrganizationProfile} from "../../database";
+import {OrganizationService} from "./organization.service";
+
+@Controller("organization")
+export class OrganizationController {
+    constructor(private readonly orgService: OrganizationService) {}
+
```
Comment (bwpikaard) [inline core/src/organization/organization/organization.controller.ts:11]: Double new line

**Review by bwpikaard — CHANGES_REQUESTED** [review]

**Review by bwpikaard — APPROVED** [review]

**Pull request merged**